### PR TITLE
edit category

### PR DESCRIPTION
**Pull request by MuhammadTamzid**

### CODE REVIEW

**Comment by sujoydatta** [comment]
ok

**Comment by SSalekin** [comment]
ok

File: app/views/admin/categories/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<h1><%= t "edit_category_text" %></h1>
+<div class="row">
+  <div class="col-md-8 col-md-offset-2">
+    <%= form_for [:admin, @category] do |f| %>
+      <%= render "shared/error_messages", object: f.object %>
+
+      <%= f.label :name %>
+      <%= f.text_field :name, class: 'form-control' %>
```
Comment (dieunb) [inline app/views/admin/categories/edit.html.erb]: double quote pls

Comment (MuhammadTamzid) [inline app/views/admin/categories/edit.html.erb]: Sorry :( . Will change

**Pull request merged**